### PR TITLE
added scale_ci_diagnosis job

### PR DESCRIPTION
**Pull request by amitsagtani97**
Added job to run the [scale-ci-diagnosis](https://github.com/openshift-scale/scale-ci-diagnosis) at the end of a job and capture the prom data.

### CODE REVIEW

**Review by mohit-sheth — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ Namespaces per cluster | Cluster Limits | Tests namespaces per cluster limit | :
 Baseline | Base/Idle cluster | Captures performance data of the base cluster with no user created objects | :heavy_check_mark: | NA | :heavy_check_mark: |
 Uperf | Networking | uperf benchmarks testing node to node, pod to pod, and svc to svc throughput and latency for TCP and UDP protocols | :heavy_check_mark: | :x: | :heavy_check_mark: |
 Kraken | Base/Idle cluster | Injects chaos scenarios into the cluster | :heavy_check_mark: | NA | :heavy_check_mark: |
+scale_ci_diagnosis | Storing Metrices | Captures Prometheus DB and cluster information using must-gather  | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark:
```
Comment (chaitanyaenr) [inline README.md:70]: s/Metrices/Metrics

**Review by chaitanyaenr — APPROVED** [review]
Minor typo, other than that LGTM.

**Comment by amitsagtani97** [comment]
@chaitanyaenr Fixed the typo :+1:

**Pull request merged**